lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/util/AzureBatchTaskJARFileName.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runime.azbatch.util;
+
+import org.apache.reef.annotations.audience.Private;
+
+/**
+ * The Azure Batch task JAR name.
+ */
+@Private
+public final class AzureBatchTaskJARFileName {
```

Comment:
Can we use a more generic name here so we may be able to add more constants as they come along to the same file? How about AzureBatchFileNames?

Reply:
Sure. I will update it.